src/main/java/bio/terra/service/filedata/DrsId.java

```diff
@@ -14,12 +13,14 @@ public class DrsId {
   private final String version;
   private final String snapshotId;
   private final String fsObjectId;
+  private final boolean compactId;
```

Comment:
I guess I tend not to name the variables that way since it gets weird with getters though we don't have getters for that one.  If it's ok, I can update in a followon (just so I don't need to rerun all the tests)

Reply:
Oh yeah no prob! And if you would prefer not to make this change, that's okay… in the wider context of the other properties `snapshotId` and `fsObjectId` (which themselves are actual ids) it was confusing to have a boolean `compactId` which was not an id.